### ISSUE TITLE
Choose Address Issue

### ISSUE THREAD

**Issue by Pokerchip28** [Hari-Nagarajan/fairgame#528]
**Describe the bug**
On the checkout page, the bot "couldn't find buttons to proceed to checkout" because it needs to select an address first. I didn't experience this issue before when testing an item.

**To Reproduce**
1. Let bot run
2. When GPU comes in stock, bot adds, then directs to checkout
3. Boom, select address Step 1, and bot freaks out.

**Expected behavior**
I expected the bot to go straight to checkout or at least click the select address button.

**Screenshots**
Screenshot attached

**Desktop (please complete the following information):**
 - OS: Windows 10 Browser: Chrome

**Additional context**
![Inkedscreenshot-ptc-error_02-26-2021_17_39_36_LI (2)](https://user-images.githubusercontent.com/16569290/109373751-104ab800-786e-11eb-8a24-b318c3a0a766.jpg)

**Comment by gtrak** [comment]
I think this is not a fairgame issue. I can't check out manually either in
this state (often I race fairgame when it finds a matching offer). It
doesn't seem like there's much selenium code can do here.

**Comment by Pokerchip28** [comment]
Gotcha. Have you had any luck with that method? lol I started trying that but no luck ;/

**Comment by gtrak** [comment]
Yea, I got 2 5700 xt's at once that way. One me, one fairgame within a
couple seconds of each other.

**Comment by DakkJaniels** [comment]
Known issue. Development branch code might click the button now (if `--shipping-bypass` is active, use at your own risk of course), but I doubt it will result in actually purchasing the product. I think this is an amazon issue, because this does not resemble the typical page flow at all.

**Comment by slider6996** [comment]
Ya i been trying to figure this issue out past few days myself with 0 luck and couldnt of happened at a worse time with all these 3070 drops but ya even doing stuff manually this is a issue i noticed so assuming something needs changed with fairgame possibly, hopefully can get worked out as whatever amazon did seems to of broke it.

**Comment by polow** [comment]
Same issue i am facing since last night, but i think this issue pops up only when item is out of stock but still shows up as available on Amazon or Amazon added this extra protection procedure to bypass bots because it's something completely new on checkout.
Hope devs can fix this

**Comment by xbryanthomasx** [comment]
Yeah, having this problem too. There was a drop and I was watching the bot and unfortunately it got stuck at the same spot @Pokerchip28 showed

**Comment by tionw** [comment]
theres a temporary fix on the development branch if you guys haven't tried it yet

**Comment by tionw** [comment]
The fix won't fix it because it will basically only do what you can do manually but faster.. the disappearing stock in checkout is an amazon issue. Also if you guys are camping for multiple things you should move the all bugged ASINs to a separate instance as the bot will get hung up on trying checkout the bugged ASINs (on repeat, until it shows OOS) without checking for your other ASINs in the meantime.

**Comment by Pokerchip28** [comment]
Can anyone confirm that the “select address” page occurs when the item is out of stock? I’m curious if it just goes out of stock and that’s the reason that the page comes up. I’ve tested with items that are in stock and it seems to work just fine. Only these GPUs ;/

**Comment by DakkJaniels** [comment]
I doubt anyone can confirm 100%, but I was trying manually and hitting the same issue... so it seems very likely.

**Comment by Pokerchip28** [comment]
Gotcha. I got the same error as well on a manual checkout and even a different Amazon account. I think it’s safe to say that the issue is on Amazon’s side.

**Comment by slider6996** [comment]
> I doubt anyone can confirm 100%, but I was trying manually and hitting the same issue... so it seems very likely.

Ya when the 3070 drops were happening i also tried manually for over like 15 mins straight and got the error

**Comment by Pokerchip28** [comment]
Wouldn’t it be most efficient if the bot pressed the “Buy Now” button instead of the add to cart button? Just a thought @Hari-Nagarajan + anyone else who knows how to modify the code lol

**Comment by DakkJaniels** [comment]
@Pokerchip28 lol well why don't you modify the bot to make it do that and submit a PR if it's so much more efficient?

**Comment by Pokerchip28** [comment]
@DakkJaniels LOL Cuz I have no idea how to write Python

**Comment by jersmo** [comment]
The buy now button resulted in the same address picking issue.

**Comment by edwin-yan** [comment]
I thought of modifying the code to click buy now first, however, I found the 'buy now' cannot garantee it is Amazon listing all the time. In addition, the Amazon listing may be in the option page which does not have buy now button at all.

**Comment by AnNoobiz** [comment]
I think I've solved this issue and I made the account to come say so. I've been trying to manually buy a card since launch with no success and yesterday I decided to finally try this bot even though I have zero experience with anything like GitHub or Python. I followed the tutorials and got it set up and was feeling pretty good, until I got stuck on the address screen. 

I tried deleting all but my primary address, which did not help. I made sure 1-click was active in all browsers, which did not help. I eliminated all but 1 payment option, which did not help. Finally I added a new BILLING address, made it the primary address, deleted my main (shipping) address then added the shipping address again, made it primary/default and deleted the billing address. 

Then I ran the bot again and my test item was ordered right away.  I really know nothing about any of this but getting down to one default address, adding a second (temporary) and making it default, deleting the original default, adding it again and deleting the temporary seemed to fix the issue for me. 

I hope this helps you, after I get my card.
